core/domain/learner_group_services.py

```diff
@@ -550,6 +600,17 @@ def remove_invited_students_from_learner_group(
     user_models.LearnerGroupsUserModel.update_timestamps_multi(models_to_put)
     user_models.LearnerGroupsUserModel.put_multi(models_to_put)
 
+    learner_group_model = learner_group_models.LearnerGroupModel.get(
+        group_id, strict=True
+    )
+    learner_group_model.invited_student_user_ids = [
+        student_id for student_id in
+        learner_group_model.invited_student_user_ids if
+        student_id not in student_ids
+    ]
+    learner_group_model.update_timestamps()
+    learner_group_model.put()
```

Comment:
Why is this added here but `remove_students_from_learner_group` did not have this edit? Can't the calling function do this removal?

Reply:
Have added this section as conditional update when required in both functions. The moto of adding this is that for certain handlers, these functions are directly called from the controller so this fetching is required in that case.